conbench/entities/run.py

```diff
@@ -165,14 +165,13 @@ def create(data) -> "Run":
     def get_baseline_run(
```

Comment:
I know this isn't part of this patch, but I wonder if we should rename this to something like `get_default_baseline_run()` so that it's (extra) clear that this isn't _the only_ baseline that could ever exist for the contender we have in `self`, it's just our best offer for a default baseline

Reply:
Yes that's a great idea. This method in general will probably become much less necessary after #384 and followup UI work. We'll be able to just hit `/api/compare/runs/run_id?target_baseline_commit=fork_point` and can skip the need for this method.